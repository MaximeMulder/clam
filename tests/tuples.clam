type A = (Int)
type B = (Int, String)

def main =
    var a: A = @(0);
    var b: B = @(1, "Hello");

    print(a);
    print(b);

    print(a.0);
    print(b.0);
    print(b.1);
    unit
