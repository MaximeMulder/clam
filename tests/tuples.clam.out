(0)
(1, "Hello")
0
1
"Hello"
